src/assets/scss/components/_bookingsSummary.scss

```diff
@@ -0,0 +1,125 @@
+@use '../abstracts' as *;
+
+.booking-summary {
+  &.fill {
+    min-height: 100vh;
+    height: 100%;
```

Comment:
this also css default height is set to 100%

Reply:
you are correct !!